Description: Disable malloc_usable_size
 Disable code introduced in sqlite 3.7.10 using malloc_usable_size, as it caused
 a regression on 64-bit platforms.
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: http://bugs.debian.org/665363

--- sqlite3-3.7.11.orig/configure.ac
+++ sqlite3-3.7.11/configure.ac
@@ -127,7 +127,7 @@
 #########
 # Figure out whether or not we have these functions
 #
-AC_CHECK_FUNCS([usleep fdatasync localtime_r gmtime_r localtime_s utime malloc_usable_size])
+AC_CHECK_FUNCS([usleep fdatasync localtime_r gmtime_r localtime_s utime])
 
 #########
 # By default, we use the amalgamation (this may be changed below...)
